src/textual/scrollbar.py

```diff
@@ -70,6 +70,13 @@ def __rich_repr__(self) -> rich.repr.Result:
 
 
 class ScrollBarRender:
+    _vertical_bars: ClassVar[list[str]] = ["▁", "▂", "▃", "▄", "▅", "▆", "▇", " "]
```

Comment:
If these are meant to be overridden, then they probably shouldn't be prefixed with an underscore. I might also put them in all-caps given they are constants to the class where they are defined.

It might also be worth considering changing them to a tuple and typing them as `tuple[str, str, str, str, str, str, str, str]`. A bit verbose, but it would allow the typechecker to mandate there are 8 items.

Reply:
Ah, in that case a list is fine!